### DISCUSSION TITLE
Documentations

### DISCUSSION THREAD

**Discussion by emreakay** [orchidsoftware/platform#1857] [Q&A] [answered]
Hi,
I want to know which software have you used to create statick doc pages on
https://orchid.software/en/docs/

Thanks.

**Comment by tabuna** [comment, marked as answer]
I used this project https://jigsaw.tighten.co/

**Reply by emreakay** [reply]
do you have any command line tool to create static docs from github repo

**Reply by tabuna** [reply]
You can see for yourself. All our code is open: https://github.com/orchidsoftware/orchid.software
